### PR TITLE
restructure Account and add NotFunded message

### PR DESCRIPTION
**Pull request by piyalbasu**
Screen Shot 2021-03-09 at 4 49 26 PM
Screen Shot 2021-03-09 at 4 49 36 PM
Screen Shot 2021-03-09 at 4 49 51 PM

### CODE REVIEW

File: extension/src/popup/components/Menu.tsx [hunk 1]
```diff
@@ -161,6 +163,7 @@ export const Menu = () => {
             </SlideoutNavListItemEl>
           </SlideoutNavListEl>
         </MenuEl>
+        <Footer />
```
Comment (piyalbasu) [inline extension/src/popup/components/Menu.tsx:166]: to make room for the new account assets UI, footer is moving here

File: extension/src/popup/components/account/AccountAssets.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
```
Comment (piyalbasu) [inline extension/src/popup/components/account/AccountAssets.tsx:1]: this is sort of placeholder. This component will be completely changed when we add an asset + payment history UI next

File: extension/src/popup/components/account/AccountDropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
+import React from "react";
```
Comment (piyalbasu) [inline extension/src/popup/components/account/AccountDropdown.tsx:1]: simply breaking this out of Account.tsx into its own component

File: extension/src/popup/components/signTransaction/Operations.tsx [hunk 4]
```diff
@@ -12,7 +12,7 @@ import { IconWithLabel, TransactionList } from "popup/basics/TransactionList";
 
 import { KeyIdenticon } from "popup/components/identicons/KeyIdenticon";
 
-import IconExcalamtion from "popup/assets/icon-exclamation.svg";
+import IconExclamation from "popup/assets/icon-exclamation.svg";
```
Comment (piyalbasu) [inline extension/src/popup/components/signTransaction/Operations.tsx:15]: 😬

**Review by quietbits — APPROVED** [review]

**Pull request merged**